tests/ert/unit_tests/config/test_forward_model.py

```diff
@@ -1011,7 +1035,9 @@ class FMWithAssertionError(ForwardModelStepPlugin):
         def __init__(self):
             super().__init__(name="FMWithAssertionError", command=["the_executable.sh"])
 
-        def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
+        def validate_pre_experiment(
+            self, fm_step_json: ForwardModelStepJSON, _: dict
```

Comment:
Very nitpick, but could this be `Dict[str, str]` instead of `dict` ?

Reply:
fixed.